### PR TITLE
Corrected skeletal creatures naming conventions

### PR DESCRIPTION
**Pull request by FulcrumA**
#### Summary
SUMMARY: Content "Corrected skeletal creatures naming conventions"

#### Purpose of change
Current "skeletal" creatures are misnamed, often leading to confusion despite their description clearly underlining these are regular (or "upgraded") zombies mutated in a way they developed additional, exoskeletal armor on top of their flesh, rather than being sciencomagically animated bones. Because of the current naming conventions, however, even tileset artists keep making mistake of taking them just for skeletons, often leading to questionable choices of those zombies' graphical representation.

#### Describe the solution
Renamed the skeletal creatures to "ossified (zombie, brute etc)".

#### Describe alternatives you've considered
Plenty of alternative, worth consideration suggestions were made. Details in down below, in comments.

#### Additional context
Aside from the creatures' names, their mentions in some related mainlined files are corrected.

### CODE REVIEW

**Comment by Mecares** [comment]
Personally i think enshelled does not sound great maybe bone carapace zombie or something else would be better?

**Comment by Maddremor** [comment]
Alternatives could be "chitinous" or "plated".

**Comment by FulcrumA** [comment]
If someone has strong feelings about these it's certainly worth consideration. So far I do personally like enshelled the most since it does not imply much about material of the 'armor' or how it's layered (and there's differences there between various creatures) while being relatively short. Bone carapace would probably describe things most accurately but it's quite longish and already may lead to some wrapped around/cut names on smaller displays in case of stuff like juggernauts.

**Comment by TechyBen** [comment]
Yeah, enshelled seems to work. Perhaps for translation ""plated" could also work as a common term, but that could be left in comments/notes for translators.

**Comment by KorGgenT** [comment]
personally i don't think enshelled is a very smooth word to use, it's very "passive" as a descriptor. will you consider one of the alternatives maddremor mentioned?

**Comment by NickGrease** [comment]
If 'ossified' translates well, that would be my choice. [Ossuary](https://s15-us2.startpage.com/wikioimage/f33b7d3545c303e535eecac95cf02e65.jpg) definitely has it's thematic advantages.

**Comment by nexusmrsep** [comment]
I'd vote for "ossified".

I like "skeletal" as it's a first connotation that comes to mind but I understand that it might be confusing. Although, is it really so unbelievable that post-apoc people wouldn't call them that?

**Comment by I-am-Erk** [comment]
I think ossified is the most accurate word here. "Plated" would get the same point across.

I agree that "enshelled" is awkward: it's not common, doesn't flow well, and isn't even all that accurate.

**Comment by FulcrumA** [comment]
@nexusmrsep 
Given that descriptions suggest that those zombies actually look nothing (or at least very much unlike) actual skeletons (since in majority of cases the flesh etc is all there, there's just carapace on top of it), I wouldn't be so sure.

Anyway, you people want "ossified"? "Ossified" I give you, due to popular demand.

**Comment by kevingranade** [comment]
First, your problem statement is weak, I don't agree that there's a problem in the first place. Names by nature are somewhat arbitrary, and particular people (tileset authors) getting confused is not evidence that there's a problem.

As for the proposed solutions, *none* of these suggestions reflect what someone would call one of these zombies upon seeing them, and most are also misleading.
Enshelled - there's no shell
chitinous - there's no chitin
"plated" - this is a bit closer, but it implies there are plates, which except for the largest examples, there are not.
"ossified" - this is the closest, but aside from being much more obscure, it's also not a descriptor of the creature as a whole.
"osseous" - as above, but even more obscure.

The reason we're still using "skeletal" is it's a very common and accessible word, and it *fits*.
These enemies are both *bony* and *spindly*, which are the primary meanings of the word "skeletal".

Confusion with a common fantasy trope isn't a reason to avoid the name, it's a reason *to* use the name. If you really found yourself in this situation and there were shambling undead walking around, and some of them were skinny and covered in bone, would you *really* avoid calling them skeletons?

**Comment by FulcrumA** [comment]
>The reason we're still using "skeletal" is it's a very common and accessible word, and it fits.

I have to confess, 'skeletal' seems to fit the least. They're not animated bones, the descriptions also do not mentions them being bony or spindly as you claim with possible exception of the dog. All the descriptions do say is that they seem to have additional ossified tissue. Which makes sense, since fluff-wise blob needs certain mass/flesh to work with to work at all.

At the very least, the "skeletal" is not fit for what the descriptions imply now, neither describing creatures as such nor underlining any difference toward regular zombies in mass (and those in turn often are being described as regular people in appearance, just with additional goo-oozing, hateful expressions and so on). Since I have no info about strong need to adjust the descriptions to fit particular vision or even there being an issue with them, I've adjusted names so they do fit those descriptions well.

**Comment by KorGgenT** [comment]
if i were a survivor and saw a huge thing with plates covering it, i'd probably call it plated or armored. especially if i'd be using the name as a device to remember the general abilities of it; ossified isn't something that's in my general usage vocabulary, but not totally outside the realm of reason. a "shell" makes me think of something like a crab or a mollusk. Skeletal heavily implies that what you're looking at doesn't really have anything that resembles "tissue, or any bulk whatsoever. Especially in today's day and age of high fantasy skeletal dragons and skeletons in most fantasy genres.

To sum up: i support the change from skeletal to something else, as the lore basically states that it's more plated, rather than lost all of its muscle mass.

**Comment by FONZACUS** [comment]
on a broader perspective, simple is the best, id go for armored/plated. my reasoning for that is because the game hasnt been fully translated to many obscure languages.

ive always wondered why skelys here could deflect gun shots where in other games theyd die from a punch. turns out they had an exoskeleton.

**Comment by Mecares** [comment]
So just rename to exoskeletal zombie?

**Comment by ifreund** [comment]
I mostly agree with Kevin here but wanted to throw out another imo less obscure/awkward option.  We could call the m “bone zombies” or “bony zombies” which is simple, accurate, and easy to understand that it’s very much not a literal walking skeleton.

**Comment by kevingranade** [comment]
Closing for lack of progress, nothing proposed significantly improves on skeletal.
Someone can look into changing the descriptions if that's a point of contention, because they ARE intended to be spindly, it's why they have HARDTOSHOOT.
